src/app/register/register.component.scss

```diff
@@ -0,0 +1 @@
+@import "../styles/components/_headings";
```

Comment:
The new lines are present, github doesn't show it here. When I click on the edit button over a file, it shows new line.